aggregator_core/src/datastore.rs

```diff
@@ -71,13 +71,19 @@ pub mod test_util;
 ///
 /// [1]: https://docs.rs/rstest_reuse/latest/rstest_reuse/
 macro_rules! supported_schema_versions {
-    ($( $i:literal ),*) => {
-        const SUPPORTED_SCHEMA_VERSIONS: &[i64] = &[$($i),*];
+    ( $i_latest:literal $(,)? $( $i:literal ),* ) => {
```

Comment:
$i_latest has to be first because `$( $i:literal ),* , $i_latest:literal` complains at the call site that the last argument is ambiguous. Fair enough.